cleaners/mc.xml

```diff
@@ -33,15 +33,15 @@
 -->
 <cleaner id="midnightcommander">
   <label>Midnight Commander</label>
-  <description>File Manager</description>
+  <description>File manager</description>
```

Comment:
Yes, lowercase is consistent with the others

Reply:
:-)